.eslintrc

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Do we want to enforce similar linting rules in the frontend and backend? I am not sure whether that's a good idea since the frameworks are so different and we are using TypeScript in the backend. What's you opinion on that @ImmanuelSegol?